<launch>

  <param name="xyz_path" value="$(find lasr_manipulation_3d_completion)/data/_0_0_5_x.xyz" />
  
  <node pkg="lasr_manipulation_3d_completion"
        type="pointcloud_completion_service.py"
        name="completion_service"
        output="screen" />

  <node pkg="lasr_manipulation_3d_completion"
        type="load_xyz_pub.py"
        name="xyz_loader"
        output="screen" />

  <node pkg="lasr_manipulation_3d_completion"
        type="call_completion_client.py"
        name="call_completion"
        output="screen" />

  <node pkg="lasr_manipulation_collision_object"
        type="add_collision_obj_service.py"
        name="completion_to_collision_service"
        output="screen" />

  <node pkg="lasr_manipulation_collision_object"
        type="add_collision_obj_client.py"
        name="insert_collision_obj"
        output="screen" />

</launch>
